lms/static/sass/views/_program-list.scss

```diff
@@ -17,6 +17,11 @@ $pl-button-color: #0079bc;
   @include span-columns(12);
 }
 .sidebar{
+    float: right !important;
```

Comment:
Why do we need the "float: right" here? Is it to make the sidebar stay on the right side if there are no program cards?